GitUI/GitModuleForm.cs

```diff
@@ -19,12 +19,12 @@ public GitUICommands UICommands
         {
             get
             {
-                if (_uiCommands == null)
-                {
-                    throw new InvalidOperationException("UICommands is null");
-                }
-
-                return _uiCommands;
+                // If this exception is seen, it's because the parameterless constructor was called.
+                // That constructor is only for use by the VS designer, and translation unit tests.
+                // Using it at run time is an error.
```

Comment:
💡 Consider marking the constructor obsolete and/or throwing an exception in the constructor if `LicenseManager.UsageMode != LicenseUsageMode.Designtime`

Reply:
👍 Thanks, this was a great tip. Marking the base ctor and subclass ctors as obsolete flagged two `GitModuleForm` subclasses that were calling down without parameters. These two forms were better fits for `GitExtensionsForm`.

The `LicenseManager` check covers the VS designer but not the unit test invocations from `TranslationTest`. I added a flag that bypasses the exception just within the scope of the `TranslationTest`.